system-security-plan.yml

```diff
@@ -0,0 +1,62 @@
+---
+name: openFEC
+uniqueID: FEC-20141001-20150915-01
+version: 1.2.0
+phase: beta
+information-types:
+- D26 Civilian Operations
+confidentiality: none
+integrity: low
+availability: low
+security-baseline: open data
+system-type: minor
+level-of-identity-assurance: 0
+staff:
+    authorizing-official:
+        name: Aaron Snow
+        title: 18F Executive Director 
+        org: General Services Administration
+        unit: 18F
+        email: 18F@gsa.gov
+    system-owner: 
+        name: Noah Kunin
+        title: 18F Infrastructure Director
+        org: General Services Administration
+        unit: 18F
+        email: devops@gsa.gov
+    system-management:
+        name: Noah Kunin
+        title: 18F Infrastructure Director
+        org: General Services Administration
+        unit: 18F
+        email: devops@gsa.gov
+    system-security-officer:
+        name: Noah Kunin
+        title: 18F Infrastructure Director
+        org: General Services Administration
+        unit: 18F
+        email: devops@gsa.gov
+    technical-lead:
+        name: Lindsay Young
+        title: Engineer
+        org: General Services Administration
+        unit: 18F
+        email: lindsay.young@gsa.gvo
+leveraged-authorizations: 
+- https://www.fedramp.gov/marketplace/compliant-systems/amazon-web-services-aws-eastwest-us-public-cloud/
+purpose: https://github.com/18F/openFEC/blob/develop/README.md
+components: 
```

Comment:
Not sure exactly what this is for, but there are also javascript requirements in the `package.json` files of the respective repos, and the project also uses the 18f/fec-cms and 18f/fec-proxy repos.